yarn-project/end-to-end/src/integration_l1_publisher.test.ts

```diff
@@ -233,6 +237,49 @@ describe('L1Publisher integration', () => {
     return Fr.fromString(entry);
   };
 
+  /**
+   * Creates a json object that can be used to test the solidity contract.
+   * The json object must be put into
+   */
+  const writeJson = (
+    fileName: string,
+    block: L2Block,
+    l1ToL2Messages: Fr[],
```

Comment:
```suggestion
    l1ToL2Messages: Fr[], // Hashes of `l1ToL2Content`
```

Had to look up what exactly `l1ToL2Content` was so having this comment here would be useful.

Reply:
Naming was just similar to what we have been calling l1ToL2Messages for the rest of the file, and it is not just the hash of the l1ToL2Content, but of the entire message (Where the content is included with other values). 

This also influences the second comment, as the `l1ToL2Content` is not the preimage, but just part of it.